pennylane/labs/resource_estimation/ops/op_math/symbolic.py

```diff
@@ -178,3 +182,114 @@ def pow_resource_decomp(
     def tracking_name(base_class, z, base_params) -> str:
         base_name = base_class.tracking_name(**base_params)
         return f"Pow({base_name}, {z})"
+
+
+class ResourceExp(Exp, re.ResourceOperator):
+    """Resource class for Exp"""
+
+    @staticmethod
+    def _resource_decomp(base_class, coeff, num_steps, **kwargs):
+
+        while isinstance(base_class, SProd):
+            coeff *= base_class.scalar
+            base_class = base_class.base
+
+        # Custom exponential operator resources:
+        if isinstance(base_class, re.ResourceOperator):
+            try:
+                return base_class.exp_resource_decomp(coeff, num_steps, **kwargs)
+            except re.ResourcesNotDefined:
+                pass
+
+        # PauliRot resource decomp:
+        if (pauli_sentence := base_class.pauli_rep) and math.real(coeff) == 0:
+            if qml.pauli.is_pauli_word(base_class):
+                num_wires = len(base_class.wires)
+                pauli_word = tuple(pauli_sentence.keys())[0]  # only one term in the sum
+                return _resources_from_pauli_word(pauli_word, num_wires)
+
+            scalar = num_steps or 1  # 1st-order Trotter-Suzuki with 'num_steps' trotter steps:
+            return _scale_dict(
+                _resources_from_pauli_sentence(pauli_sentence), scalar=scalar, in_place=True
+            )
+
+        raise re.ResourcesNotDefined
+
+    def resource_params(self):
+        return {
+            "base_class": self.base,
+            "coeff": self.scalar,
+            "num_steps": self.num_steps,
+        }
+
+    @classmethod
+    def resource_rep(cls, base_class, coeff, num_steps, **kwargs):
+        name = f"Exp({base_class.__class__.__name__}, {coeff}, num_steps={num_steps})".replace(
```

Comment:
Lets use the tracking name here. This change will require `base_params` like I commented earlier. It should look something like this.
```
base_name = base_class.tracking_name(**base_params)
name = f"Exp({base_name}, ..."
```